app/models/tutoring_session.rb

```diff
@@ -2,9 +2,11 @@ class TutoringSession < ApplicationRecord
   belongs_to :student
   belongs_to :tutor
 
-  validates :location,     presence: true
-  validates :start_time,   presence: true
-  validates :end_time,     presence: true
+  validates :location, presence: true
+  validates :hours, presence: true,
+                    length: { maximum: 4 }
```

Comment:
maximum 4 hours per month? If you tutor every Sunday, does this mean that months with 5 Sundays can still only have 4 hours per month?